articles/data-binding-with-angular/index.md

```diff
@@ -0,0 +1,301 @@
+**Angular** is a platform and design framework for building a SPA using HTML and TypeScript.
+ Angular is written in TypeScript(although the first version, i.e.AngularJS, was written in JS).
+ It implements its core and optional functionalities as a set of TypeScript libraries that 
```

Comment:
Let's make sure we are explaining things so that a beginner can grasp them. 
This sentence is straight from the docs - it would be nice to expand and explore what all this means as you guide the audience along.